### PR TITLE
all the tags updated with the storybook react common library

### LINKED ISSUE #38: Update React Commons Collection Docs
Need to update documentation for React Commons Collection.

### PR DESCRIPTION
**Pull request by dawoodali16**
## React Pakistan Docs

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-pakistan/react-pakistan-docs/blob/dev/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-pakistan/react-pakistan-docs/blob/dev/CONTRIBUTING.md)
- [ ] Double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-pakistan/react-pakistan-docs/blob/dev/LICENSE.md).

### CODE REVIEW

File: docs/react-commons-collection.md [hunk 1]
```diff
@@ -37,12 +37,12 @@ Add React Commons Collection base theme to your project.
...
-  import { theme } from '@react-pakistan/util-functions';
+import { ThemeProvider } from 'styled-components';
+import { theme } from '@react-pakistan/util-functions';
 
-  <ThemeProvider theme={theme}>
-    <App />
-  </ThemeProvider>
+<ThemeProvider theme={theme}>
+  <App />
+</ThemeProvider>;
```
Comment (Taimoormk) [inline docs/react-commons-collection.md]: Please remove the semicolon.

File: docs/react-commons-collection.md [hunk 2]
```diff
@@ -57,29 +57,30 @@ We aim to achieve 100% coverage and we shall...
...
+  mobileM=''
+  mobileS=''
+  referrerpolicy=''
+  rel=''
+  tablet=''
+  target='_blank'
+  type=''
+>
+  Go to Google
+</A>;
```
Comment (Taimoormk) [inline docs/react-commons-collection.md]: Remove semicolons from all of these instances.

**Review by Taimoormk — CHANGES_REQUESTED** [review]
Mostly it is ok, there are 3 areas which need adjustments:

- You have added a semicolon after when every component ends, we don't need a semicolon there, please remove them.
- You removed the indentation from all components, please revert it.
- Need to alphabetically sort all props for all components, download VSCode extension https://github.com/amatiasq/vsc-sort-imports.git and use it to automatically sort all components props.

Rest is alright, please do these changes and we can merge this PR.

**Comment by dawoodali16** [comment]
Sir, react-commons-collection is now updated, you can check now, I didn't understand here, how to make the pull request that's why I did not make the pull request here, please check and now I start working on the other one . Thanks

**Review by dawoodali16 — COMMENTED** [review]
@Taimoormk Sir, please check the indentation of {SocialShare}, is this one is fine or not ?

**Review by Taimoormk — CHANGES_REQUESTED** [review]
@dawoodali16 I am afraid the indentation is still not quite right, this is taking too much of time now, so I will fix this later tonight, please continue with the second assigned issue.

**Comment by dawoodali16** [comment]
Sir, now the indentation is cleared and there is only issue with the double quote mark
@Taimoormk  Sir, please check the latest commit, hope there is much difference with other one

**Pull request merged**